ppoprf/src/ppoprf.rs

```diff
@@ -84,12 +85,33 @@ impl ProofDLEQ {
 pub type ServerPublicKey = Vec<RistrettoPoint>;
 
 // The wrapper for PPOPRF evaluations (similar to standard OPRFs)
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize)]
 pub struct Evaluation {
+    #[serde(serialize_with = "ristretto_serialize")]
     output: CompressedRistretto,
     proof: Option<ProofDLEQ>,
 }
 
+#[derive(Deserialize)]
+pub struct Points(
+    #[serde(deserialize_with = "ristretto_deserialize")]
+    CompressedRistretto
```

Comment:
It's unfortunate we have to wrap this just to attach a serde adaptor. Still, seems like an improvement.

Reply:
yeah, still much better than implementing the Serialize/Deserialize traits!